Dockerfile

```diff
@@ -54,7 +54,7 @@ USER oioioi
 ENV PATH $PATH:/home/oioioi/.local/bin/
 
 ENV BERKELEYDB_DIR /usr
-RUN pip3 install --user psycopg2-binary==2.8.6 twisted uwsgi
+RUN pip3 install --user psycopg2-binary==2.9.10 twisted uwsgi
```

Comment:
Could you explain a bit why this version bump is required?

Reply:
Actually, this version bump isn't required. Initially, there was a plan to upgrade to Python 3.11, but on Python 3.10, psycopg 2.8.6 works well.